.github/workflows/check-pr-title.yaml

```diff
@@ -0,0 +1,20 @@
+name: "Check PR title"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if the PR title is well dressed
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+        run: |
+          echo "$PR_TITLE" | grep -E '^(((build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\([\w\-\.]+\))?(!)?)|((ROX-[0-9]+,)*ROX-[0-9]+))(: .*)$';
+          if [ "$(echo "$PR_TITLE" | wc --chars)" -gt 100 ]
```

Comment:
https://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting#comment64271489_11993051 says

> Github will hide commit message text after the 70th character. 

This seems to match my experience when I look at https://github.com/stackrox/stackrox/commits/master

There's a value to force shorter first line.